node/coinstacks/solana/api/src/service.ts

```diff
@@ -0,0 +1,38 @@
+import type { Account, API, Tx, TxHistory } from './models'
+import { BaseAPI, handleError,  SendTxBody } from '@shapeshiftoss/common-api'
+import { Helius } from "helius-sdk";
+
+export interface ServiceArgs {
+  rpcUrl: string
+  rpcApiKey: string
+}
+
+export class Service implements Omit<BaseAPI, 'getInfo'>, API {
+  private readonly heliusSdk: Helius
+
+  constructor(args: ServiceArgs) {
+    this.heliusSdk = new Helius(args.rpcApiKey)
+  }
+
+  async getTransaction(): Promise<Tx> {
+    throw new Error('Method not implemented.')
+  }
+
+  async getAccount(): Promise<Account> {
+    throw new Error('Method not implemented.')
+  }
+
+  async getTxHistory(): Promise<TxHistory> {
+    throw new Error('Method not implemented.')
+  }
+
+  async sendTx(body: SendTxBody): Promise<string> {
```

Comment:
i wish I would have use `body.rawTx` or something more generic since `hex` doesn't really make sense for solana... can make a breaking change to update or just leave as is for now.

Reply:
For the sake of simplicity I would keep it as it is, we don't necessary need to change it now but in a followup or another ticket related to this

I would prefer to have the diff for the send endpoint in this commit and another commit for the refactoring so in case any thing occur it's simpler to revert or investigate

Reply:
Agreed, sounds like a plan